pipelines/deployment.yml

```diff
@@ -2115,6 +2178,34 @@ jobs:
         UAA_CLIENT_ID: 'response_operations'
         UAA_CLIENT_SECRET: ((latest_response_operations_client_secret))
 
+- name: response-operations-social-ui-latest-deploy
+  serial_groups: [response-operations-social-ui-latest-deploy]
+  plan:
+  - get: response-operations-social-ui-source
+    trigger: true
+    passed: [ci-rasrm-acceptance-tests]
+  - put: create-redis
+    resource: cf-cli-resource-latest
+    params:
+      command: create-service
+      service: elasticache-broker
+      plan: small
+      service_instance: ras-redis
+      timeout: 1800
+      wait_for_service: true
+  - put: cf-resource-latest
+    params:
+      current_app_name: response-operations-social-ui-concourse-latest
+      manifest: response-operations-social-ui-source/manifest.yml
+      path: response-operations-social-ui-source
+      environment_variables:
+        <<: *latest_security_user
+        <<: *latest_service_endpoints_for_python
+        RAS_SECURE_MESSAGING_JWT_SECRET: ((latest_jwt_secret))
```

Comment:
Is `RAS_SECURE_MESSAGING_JWT_SECRET` used in this app?

Reply:
Deleted.